config/cookstyle.yml

```diff
@@ -920,6 +920,13 @@ ChefEffortless/SearchForEnvironmentsOrRoles:
   Exclude:
     - '**/metadata.rb'
 
+ChefEffortless/Berksfile:
+  Description: Policyfiles should be used for cookbook depsolving instead of a Berkshelf Berksfile.
```

Comment:
is `depsolving` an actual word? 🤔 cc/ @mjingle

Reply:
We use it in Cheflandia a lot

Reply:
@afiune Supposedly it is, but it is an informal technical term. After doing a deep, deep dive search,`depsolving` supposedly means "dependencies solving." I would encourage using a less obscure, and less technical wording instead.

Reply:
Updated to dependency solving